setup.cfg

```diff
@@ -125,6 +126,7 @@ all =
     apache-airflow-providers-google>=8.1.0
     apache-airflow-providers-http
     apache-airflow-providers-snowflake
+    snowflake-sqlalchemy>=1.4.4  # Temporary solution for https://github.com/astronomer/astronomer-providers/issues/958, we should pin apache-airflow-providers-snowflake version after it pins this package to great than or equal to 1.4.4.
```

Comment:
Can we move this line beneath the `protobuf` dependency as the order in this `all` section looks like to be alphabetically sorted?

Reply:
sure. just update it!